### ISSUE TITLE
Replace compatibility layer in tracing

### ISSUE THREAD

**Issue by aoudiamoncef** [massalabs/massa#4478]
In the context of the `tracing` crate in Rust, `tracing::debug` and `tracing::log::debug` refer to different macros used for emitting log events. Here's the difference between them:

1. **`tracing::debug!`**: This macro is provided directly by the `tracing` crate. It allows you to emit a log event with the `debug` level. You can use it like this:

   ```rust
   tracing::debug!("This is a debug log message");
   ```

   With this macro, you are using the `tracing` crate's native logging capabilities directly.

2. **`tracing::log::debug!`**: This macro is part of the `tracing-log` crate, which is a compatibility layer between the `log` crate and the `tracing` ecosystem. If you want to bridge the gap between traditional `log`-based logging and `tracing`, you can use `tracing-log` and its macros. With `tracing-log`, you can continue using the familiar `log` macros while leveraging `tracing`'s structured and contextual logging capabilities.

   ```rust
   tracing_log::log::debug!("This is a debug log message using tracing-log");
   ```

   Using `tracing-log::log::debug!` allows you to integrate `log`-style logging into your `tracing` instrumentation. This can be useful if you have existing code or libraries using the `log` crate and want to transition gradually to `tracing`.

In summary, if you are working exclusively with `tracing` and want to take advantage of its full capabilities, you can use `tracing::debug!`. If you are integrating `tracing` into an existing codebase that heavily uses the `log` crate, or if you want to bridge the gap between `log`-style logging and `tracing`, you can use `tracing-log::log::debug!`.
